src/main/java/com/minecolonies/commands/ListCitizens.java

```diff
@@ -0,0 +1,171 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.IColony;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.Style;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.event.ClickEvent;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.*;
+
+/**
+ * List all colonies.
+ */
+public class ListCitizens extends AbstractSingleCommand
+{
+
+    private static final String ID_TEXT                = "§2ID: §f";
+    private static final String NAME_TEXT              = "§2 Name: §f";
+    private static final String COORDINATES_TEXT       = "§2Coordinates: §f";
+    private static final String LIST_COMMAND_SUGGESTED = "/mc citizens list ";
+    private static final int    CITIZENS_ON_PAGE       = 9;
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public ListCitizens(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        int page = 1;
+        int colonyId = 1;
+
+        final IColony tempColony = ColonyManager.getIColonyByOwner(sender.getEntityWorld(), sender.getCommandSenderEntity().getUniqueID());
+        if(tempColony != null)
+        {
+            final Colony colony = ColonyManager.getColony(sender.getEntityWorld(), tempColony.getCenter());
+            if(colony != null)
+            {
+                colonyId = colony.getID();
+            }
+        }
+
+        if (args.length != 0)
+        {
+            if(args.length >= 2)
+            {
+                try
+                {
+                    colonyId = Integer.parseInt(args[0]);
+                    page = Integer.parseInt(args[1]);
+                }
+                catch (NumberFormatException e)
+                {
+                    //ignore and keep page 1.
+                }
+            }
+            else
```

Comment:
if we keep this style I have to do something about argumentparsing... thats looks horrible 😆 Will do that after the other two